app/controllers/webhooks_controller.rb

```diff
@@ -227,6 +227,10 @@ def generate_seller_gift_card_id(seller_id:)
     raise CannotGenerateUniqueHash, 'Error generating unique gift_card_id'
   end
 
+  def generate_seller_gift_card_id_hash
```

Comment:
This also doesn't look it like should be a part of this PR hmmm

Reply:
ahhhhhhhhhhhhhh